nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
```

Comment:
@olegz yes it is just for display. It essentially was put in place to prevent servers from exposing names that the dataflow creator may prefer to omit. 

There are other use cases tho:

- Load balanced SMTP instances
- Shared TLS certificates (where TLS validation is used from the sending MTA side)
- "Please don't share my internal hostnames"

and so on